code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -279,7 +279,7 @@
 	animate(src, alpha = 255, time = 10, flags = ANIMATION_END_NOW) //we may have a previous animation going. finish it first, then do this one without delay.
 	sleep(10)
 //as long as they're still on the sigil and are either not a servant or they're a servant AND it has remaining vitality
-	while(L && (!is_servant_of_ratvar(L) || (is_servant_of_ratvar(L) && (GLOB.ratvar_awakens || GLOB.clockwork_vitality))) && get_turf(L) == get_turf(src))
+	while(L && (!is_servant_of_ratvar(L) || (is_servant_of_ratvar(L) && (GLOB.ratvar_awakens || GLOB.clockwork_vitality))) && get_turf(L) == get_turf(src) || !L.buckled)
```

Comment:
uh, what are you doing? this will short-circuit to activating if they aren't buckled at all, regardless of actual status.

Reply:
that was meant to be an &&, dang it!

Reply:
Also, if it's meant to specifically prevent impaling, then make it check for specifically that so that idiots with rollerchair-cuffed crew don't faff around wondering why it doesn't work.
And **put it in the changelog**

Reply:
This change actually slipped my mind completely when I was writing the PR body, so I'll put that in now.